lib/assessment.rb

```diff
@@ -5,35 +5,52 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
```

Comment:
Yep, this is right! Pro-tip: you can actually write all those `attr_reader` calls in a single line, like this:
`attr_reader :name, :breed, :age`